MaxText/prefill_packing.py

```diff
@@ -293,38 +277,33 @@ def zero_padded(arr: list[int], padding: int):
       prefill_result.append((first_tokens[i], bucket.slots[i]))
     return prefill_result, decode_state
 
-  def _process_batch_compiled(
-    self,
-    params: Params,
-    padded_length: int,
-    capacity: int,
-    num_prompts: int
-  ):
+  def _process_batch_compiled(self, params: Params, padded_length: int, capacity: int, num_prompts: int):
     """Ahead-of-time compilation wrapper of _process_batch()."""
 
     if (padded_length, num_prompts) not in self.process_batch_func:
-      log.info(f"compile prefill process_batch{(padded_length, num_prompts)} {capacity=}")
+      log.info("compile prefill process_batch{(%d, %d)} %d",
```

Comment:
This should be `capacity=%d` since `{capacity=}` gets expanded to that in an f-string

Reply:
Good find; fixed